lib/carin_for_blue_button_test_kit/docs/carin_for_blue_button_v2.0.0_client_suite_description.md

```diff
@@ -0,0 +1,106 @@
+The CARIN for Blue Button v2.0.0 Client test suite verifies the conformance of client systems to the STU2 HL7® FHIR®
+[CARIN for Blue Button® Implementation Guide](http://hl7.org/fhir/us/carin-bb/STU2). These tests are intented to validate 
+CARIN IG for Blue Button Requestors, or applications that initiate a data access request to retrieve patient data.
+This can be thought of as the client in a client-server interaction.
+
+Any actor acting a FHIR Client in this IG SHALL:
+  - Be able to process and retain all profile data elements that have a minimum cardinality >= 1 and/or flagged
+  as Must Support as defined by that profiles StructureDefinition
+  - Conform to the US Core Client Capability Statement expectations for that profiles type
+
+## Scope
+
+These tests are a **DRAFT** intended to allow implementers to perform
+preliminary checks of their systems against the requirements stated for CARIN IG for Blue Button client actors
+and [provide feedback](https://github.com/inferno-framework/subscriptions-test-kit/issues)
+on the tests. Future versions of these tests may verify other
+requirements and may change the test verification logic.
+
+## Test Methodology
+
+For these tests Inferno simulates a Carin for Blue Button server application. Inferno will wait for
+the client under test to request a resource for each of the Carin for Blue Button profiles, which include the Patient,
+Coverage, Inpatient Institutional ExplanationOfBenefit, Oral ExplanationOfBenefit, Outpatient Institutional ExplanationOfBenefit,
+Pharmacy ExplanationOfBenefit, Professional NonClinician ExplanationOfBenefit, Organization, Practitioner, and RelatedPerson profiles.
+The tests also waits for search requests to be made for each of the required searches described for each resource in the
+Carin for Blue Button IG and the US Core 3.1.1 IG. For each request sent in from the client, Inferno will forward the
+request to the Inferno Reference Server, where it will grab the appropriate Carin for Blue Button resources to send back to the
+client. Each of these resources will contain all of the required and Must Support elements specified by the Carin for Blue Button IG
+to ensure the client can process a resource containing all of these fields.
+
+Once the client has finished submitting requests, Inferno will verify that the client has requested and received the correct
+resources from the Inferno Reference server for each of the required Carin for Blue Button profiles, and it will verify that a
+search request was made for each of the required searches listed for each resource. Using the interactions between Inferno and the
+client under test, Inferno will verify that the client meets the obligations from the implementation guide by ensuring the client is
+able to request and receive all the required resource profiles.
```

Comment:
I think we can clean this up a bit. Each of the two sentences seem to me to be saying about the same thing.

Proposal:

>Once the tester has finished submitting requests to Inferno from the client under test, Inferno will review the requests performed verify conformance. Specifically, Inferno checks that the client
>- Retrieved instances of each [CARIN IG for Blue Button profile](https://hl7.org/fhir/us/carin-bb/STU2/artifacts.html#structures-resource-profiles)
>- Performed searches using the search parameters and search parameter combinations marked as **SHALL** within the [C4BB CapabilityStatement](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html) and the [US Core Client CapabilityStatement](https://hl7.org/fhir/us/core/STU3.1.1/CapabilityStatement-us-core-client.html).